plutus-benchmark/nofib/test/9.6/queens4-bt.eval.golden

```diff
@@ -1,7 +1,7 @@
-CPU:             3_372_957_578
-Memory:             18_765_469
-Term Size:               1_781
-Flat Size:               1_760
+CPU:             3_363_965_578
+Memory:             18_709_269
+Term Size:               1_791
+Flat Size:               1_764
```

Comment:
Any idea why sometimes both term size and flat size get bigger?

Reply:
having extra CSE passes seems to increase the term size. I'm not entirely sure.